devito/types/dimension.py

```diff
@@ -107,6 +107,8 @@ class Dimension(ArgProvider):
     is_Incr = False
     is_Block = False
 
+    indirect = False
```

Comment:
this may need some work

Reply:
It's jus a Dimension class property the same as all the `is_...` above

Reply:
prod